CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Changelog
 
+### 2.0.0
+
+- changes `Provider` API to accept store as a prop instead of context.
```

Comment:
What you think about, 

Changing Provider API to accept store as a prop instead of context.